dom0-updates/qubes-dom0-update

```diff
@@ -96,20 +107,21 @@ if [ "$YUM_ACTION" == "reinstall" ] || [ "$YUM_ACTION" == "upgrade" ] || [ "$YUM
         exit 1
     fi
 elif [ "$YUM_ACTION" == "search" ] || [ "$YUM_ACTION" == "info" ]; then # No need to shutdown for search/info
-    TEMPLATE_EXCLUDE_OPTS=""
+    TEMPLATE_EXCLUDE_OPTS=()
 else
-    TEMPLATE_EXCLUDE_OPTS="--exclude=`rpm -qa --qf '%{NAME},' qubes-template-\*|head -c -1`"
+    TEMPLATE_EXCLUDE_OPTS=( "--exclude=$(rpm -qa --qf '%{NAME},' qubes-template-\*)" )
```

Comment:
Dropping trailing coma is very important. See https://github.com/QubesOS/qubes-core-admin-linux/commit/d7c23e1b7fdaf307138851ba24cd242ee81c4001

Reply:
I am sorry. I missed this while rebasing my changes from an older version. I will address this issue in a moment.

Reply:
This should now be resolved. Thank you for your patience.